app/src/main/java/com/r5k/contacerveja/ui/main/presenter/MainPresenter.kt

```diff
@@ -99,9 +99,19 @@ class MainPresenter<V:MainMVPView, I : MainVMPInteractor> @Inject internal const
             val drinkIdIn = withContext(context = Dispatchers.IO) {
                 interactor!!.addDrink(drinkName).await()
             }
-            Log.d(TAG,"addNewDrink drinkIdInBd="+drinkIdIn)
-            getView()!!.loadDrink(drinkIdIn)
+            Log.d(TAG,"addNewDrink drinkIdInBd=$drinkIdIn")
+            getView()!!.addNewDrink(drinkIdIn)
 
         }
     }
+
+    override fun changeDrinkAmount(drink: Drink, amount: Int) {
+        GlobalScope.launch(context = Dispatchers.Main) {
+            val updatedDrink = withContext(context = Dispatchers.IO) {
+                interactor!!.changeDrinkAmount(drink, amount).await()
+            }
+            Log.d(TAG,"changeDrinkAmount from=${drink.qnt} to=${updatedDrink.qnt}")
+            loadDrinks()
```

Comment:
Isn't necessary update all bills and drinks, we can update only the value of amount.